user-interface/src/lib/components/uswds/Banner.scss

```
@@ -2,4 +2,22 @@
   .usa-banner__inner {
     max-width: none;
   }
+  .development {
+    background-color: #419ce1;
+    color: white;
+  }
+
+  .staging {
```

The environment name in LaunchDarkly is `test`.
```suggestion
  .test {
```